BadgeBLE.md

```diff
@@ -50,3 +58,130 @@ The "mode" bytes are a combination of two 4 bit values. The high nibble describe
 | 0x07 | "picture"         |
 | 0x08 | "laser"           |
 
+### Next-Gen profile
+
+ 16-bit Service/characteristic:
+
+- Service-UUID: 0xF055 (128-bit equivalent: 0000f055-0000-1000-8000-00805f9b34fb)
+- Characteristic: 0xF056 (128-bit equivalent: 0000f056-0000-1000-8000-00805f9b34fb)
+  - Write property: to receive controls and configs
+  - Notify property: to return error codes
+  - Read property: to be implemented
+
+#### Protocol
+
+The next-gen characteristic takes message varies in length (0 - 512 bytes). The first byte is function/command code and the rest of the message is parameters for the corresponding function/command.
+
+![](/assets/ng-protocol.svg)
+
+Supported functions/commands:
+
+- power_setting
+- streaming_setting
+- stream_bitmap
+- ble_setting
+- flash_splash_screen
+- save_cfg
+- load_fallback_cfg
+
+Client app should enable notify for the characteristic to receive the returned error code. e.g. setCharacteristicNotification() for Android.
+
+##### power_setting
+
+Function/Command code: 0x01
+
+Parameters:
+
+- Power off: 0x00
+- Enable resetting after uploading done: [0x01, 0x00]. Note: call [save_cfg](#save_cfg) command to save this config.
+- Disable resetting after uploading done: [0x01, 0x01]. Note: call [save_cfg](#save_cfg) command to save this config.
+- Power off: 0x02
+
+Returns:
+
+- Parameters out of range: 0xff
+- Success: 0x00
+
+##### streaming_setting
+
+Function/Command code: 0x02
+
+Parameters:
+
+- Enter streaming mode: 0x00. This will stop all animation, clear the screen, and enter streaming mode.
```

Comment:
Enter streaming mode (0x00). This command stops all animations, clears the screen, and switches the device to streaming mode.